stable/mariadb/README.md

```diff
@@ -45,21 +45,22 @@ The command removes all the Kubernetes components associated with the chart and
-| -----------------------     | ----------------------------------         | ---------------------------------------------------------- |
-| `image`                     | MariaDB image                              | `bitnami/mariadb:{VERSION}`                                |
-| `imagePullPolicy`           | Image pull policy.                         | `IfNotPresent`                                             |
-| `mariadbRootPassword`       | Password for the `root` user.              | `nil`                                                      |
-| `mariadbUser`               | Username of new user to create.            | `nil`                                                      |
-| `mariadbPassword`           | Password for the new user.                 | `nil`                                                      |
-| `mariadbDatabase`           | Name for new database to create.           | `nil`                                                      |
-| `persistence.enabled`       | Use a PVC to persist data                  | `true`                                                     |
-| `persistence.existingClaim` | Use an existing PVC                        | `nil`                                                      |
-| `persistence.storageClass`  | Storage class of backing PVC               | `nil` (uses alpha storage class annotation)                |
-| `persistence.accessMode`    | Use volume as ReadOnly or ReadWrite        | `ReadWriteOnce`                                            |
-| `persistence.size`          | Size of data volume                        | `8Gi`                                                      |
-| `resources`                 | CPU/Memory resource requests/limits        | Memory: `256Mi`, CPU: `250m`                               |
-| `config`                    | Multi-line string for my.cnf configuration | `nil`                                                      |
+|          Parameter          |                Description                 |                   Default                   |
+| --------------------------- | ------------------------------------------ | ------------------------------------------- |
+| `image`                     | MariaDB image                              | `bitnami/mariadb:{VERSION}`                 |
+| `imagePullPolicy`           | Image pull policy.                         | `IfNotPresent`                              |
+| `usePassword`               | Enable password authentication             | `true`                                      |
+| `mariadbRootPassword`       | Password for the `root` user.              | Randomly generated                          |
+| `mariadbUser`               | Username of new user to create.            | `nil`                                       |
+| `mariadbPassword`           | Password for the new user.                 | `nil`                                       |
```

Comment:
@prydonius I mean that, when using the chart first time, as a requirement of another parent-chart, the mysql/percona/mariadb READMEs don't tell you the names of the password-secret-keys you need to put into your Pod/Deployment and/or your parent software component. It's only after actually trying to initially deploying the mariadb chart on kubernetes (but not being able to connect to it, since the software won't yet be configured with the names of the secrets) that you find out what secret key names you should have used, via the NOTES.txt output you only see at that point. Manually extracting the secret password values after-the-fact isn't very convenient when using parent/child charts.